package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "lecture-08-node-api",
+  "version": "1.0.0",
+  "description": "401 JS --  Lab 08 rest api using http module ===",
+  "main": "index.js",
+  "scripts": {
+    "test": "./node_modules/mocha/bin/mocha",
+    "debug": "DEBUG=note* ./node_modules/mocha/bin/mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/thegrimheep/lecture-08-node-api.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/thegrimheep/lecture-08-node-api/issues"
+  },
+  "homepage": "https://github.com/thegrimheep/lecture-08-node-api#readme",
+  "dependencies": {
```

Comment:
You need to ``npm i -S dotenv``. I got an error running your test because it wasn't installed as a dependency.